integrationTests/cypress/integration/redmine_administration_rest.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Administrative Procedures via API
+
+  @requires_testuser
+  Scenario: cas user (admin) => access users.json with api key => success
```

Comment:
Suggestion:
"cas user (admin) => access users.json with api key => success" -->
"a cas user which is assigned to the admin user-group access the users api endpoint successfully"